kubeflow/trainer/backends/training_runtime_loader.py

```diff
@@ -0,0 +1,657 @@
+# Copyright 2025 The Kubeflow Authors.
```

Comment:
Can you move this file to the `backend/container/training_runtime_loader.py` for now ?